packages/nc-gui-v2/package.json

```diff
@@ -14,6 +14,7 @@
     "jwt-decode": "^3.1.2",
     "nocodb-sdk": "file:../nocodb-sdk",
     "socket.io-client": "^4.5.1",
+    "vite-plugin-vuetify": "^1.0.0-alpha.12",
```

Comment:
What's this for? :)

Reply:
Sorry, I forgot to remove it... will remove it